doc/source/multivisit_products.rst

```diff
@@ -0,0 +1,243 @@
+by HST.
+
+The best option for anyone wanting to explore the region affected by such observations would be to get the list of
+exposures that contribute to the MVM mosaic and reprocess the SkyCell interactively without including the problematic
+exposures.  Eventually, these observations may be removed from the archived products, but until that time comes,
+manual reprocessing would be the suggested means for studying SkyCells affected by such exposures.
+
+
+Alignment Accuracy Across the SkyCell
+--------------------------------------
+Every effort gets made to align all exposures to a GAIA-based coordinate system.
+Some exposures, though, just can not be aligned (or aligned in an automated fashion) to the GAIA system due to any
+number of reasons.  In addition, not all exposures can be aligned to the same GAIA-based catalog of sources as the
+exposure may not include enough GAIA sources for alignment, but may contain sources measured by other projects (like
+Pan-STARRS) that have been subsequently fit to the GAIA coordinate system.  This will result in a larger uncertainty
+for the coordinates for those sources.  The best available WCS for any given exposure eventually gets defined by the
+SVM processing performed by the HST calibration pipeline and these aligned products then get used as inputs to generate
+the MVM mosaics.
+
+MVM mosaics include as many exposures as possible but due to these reasons, a given MVM mosaic can contain exposures
+fit to different astrometric catalogs.  Exposures which do not overlap other exposures in the MVM mosaic can only use
+the WCS defined during SVM processing and that may not be the most accurate GAIA-based catalog available.  This can result
+in errors in the relative alignment between those exposures and the rest of the exposures in the mosaic which have been
+aligned to the most accurate catalog available.  These errors may not be large (less than a few pixels in nearly all cases),
+but care must be taken to interpret the positions of sources across a SkyCell due to such effects.
+
+
+Effects of Proper Motion
+-------------------------
+HST has been taking images for over 30 years, with ACS being active since 2002 and WFC3 having been installed in 2009.
+As a result, there have been some fields which have been observed multiple times over those decades allowing HST with
+it's high resolution to observe the stellar proper motions.  Any source with proper motions of a few milli-arcseconds
+motion per year or more could potentially be observed moving when comparing images taken over the life of HST.  Such
+sources can be found in observations of sources like nearby globular clusters (e.g., 47Tuc or Omega Cen) or open clusters
+(like M35).
+
+Unfortunately, this makes aligning images of such high proper motion sources difficult to interpret.  Measurements of
+each sources proper motions have been included in the catalog information for sources in the GAIA-based astrometric
```

Comment:
Suggest change "Measurements of each sources proper motions have" ==> "Measurements of proper motions for each source have".

Reply:
Fixed in 175ee20.